Atomic/atomic.py

```diff
@@ -763,12 +773,21 @@ def get_col_lengths(_images):
             return
 
         _images = self.get_images()
+
+        used_image_ids = [x['ImageID'] for x in self.get_containers()]
+
         if len(_images) >= 0:
             _max_repo, _max_tag = get_col_lengths(_images)
-            col_out = "{0:" + str(_max_repo) + "} {1:" + str(_max_tag) + \
-                      "} {2:12} {3:19} {4:10}"
-            self.write_out(col_out.format("REPOSITORY", "TAG", "IMAGE ID",
-                                         "CREATED", "VIRTUAL SIZE"))
+            col_out = "{0:3} {1:3} {2:" + str(_max_repo) + "} {3:" + str(_max_tag) + \
+                      "} {4:14} {5:18} {6:14}"
+            self.write_out("")
+            self.write_out(col_out.format(Highlight.BOLD + "",
+                                          " ",
```

Comment:
Only do this if 
```
  if sys.stdout.isatty():
```
Otherwise you end up with stuff that looks like:
```
ESC[1m      REPOSITORY         TAG      IMAGE ID       CREATED            VIRTUAL SIZE 
ESC[1m   >  docker.io/fedora   latest   sha256:ddd5c   2016-03-04 13:40   204.69 MBESC[0m 

```